airflow/contrib/operators/awsbatch_operator.py

```diff
@@ -75,7 +76,7 @@ def __init__(self, job_name, job_definition, job_queue, overrides, max_retries=4
         self.job_queue = job_queue
         self.overrides = overrides
         self.max_retries = max_retries
-
+        self.execution_timeout = kwargs.get('execution_timeout', timedelta(seconds=3600))
```

Comment:
I would suggest not pulling this from the `kwargs` and make it an argument with a default value. WDYT?